stubs/pycocotools/pycocotools/cocoeval.pyi

```diff
@@ -53,7 +64,7 @@ class Params:
     recThrs: _NDArray
     # recThrs: npt.NDArray[np.float64]
     maxDets: list[int]
-    areaRng: list[int]
+    areaRng: list[list[int | float]]
```

Comment:
Nit: For typing purposes, `float` is a super-type of `int`, so the following is identical and preferred:

```suggestion
    areaRng: list[list[float]]
```